src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -189,9 +222,8 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             new object[] { Configuration.Http.WrongHostNameCertRemoteServer },
         };
 
-        [ActiveIssue(7812, TestPlatforms.Windows)]
         [OuterLoop] // TODO: Issue #11345
-        [Theory]
+        [ConditionalTheory(nameof(ClientSupportsDHECipherSuites))]
         [MemberData(nameof(CertificateValidationServers))]
         public async Task NoCallback_BadCertificate_ThrowsException(string url)
```

Comment:
Please refer to #7812.  This is prevent CI failures.  We don't want these tests running on Windows OS versions that don't have the fixes to the DHE handshakes bugs.  It is not a feature we want to test, per se.